### PR TITLE
feat: bump vega typings to 1.0

### PR DESCRIPTION
**Pull request by domoritz**
This will make it easier to release patch versions of the typings.

### CODE REVIEW

**Comment by mcnuttandrew** [comment]
Looks like 5.1 breaks ?? for older node versions (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-1.html). This may necessitate adding a minimum engine version

File: packages/vega-typings/package.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
   "dependencies": {
     "@types/geojson": "7946.0.4",
     "vega-event-selector": "^3.0.1",
-    "vega-expression": "^5.0.1",
-    "vega-util": "^1.17.1"
+    "vega-expression": "^5.1.0",
```
Comment (mcnuttandrew) [inline packages/vega-typings/package.json:24]: did you mean to version these as well?

Comment (domoritz) [reply packages/vega-typings/package.json:24]: Yeah, I think otherwise this leads to issues like #3771

**Comment by mcnuttandrew** [comment]
Per outside discussion, Lgtm

**Pull request merged**